go.mod

```diff
@@ -147,6 +147,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/Shopify/toxiproxy/v2 v2.5.0
```

Comment:
nit: embed this within the require block instead of being separated out

Reply:
Done